docker/README.md

```diff
@@ -0,0 +1,68 @@
+# 🐳 Docker Compose for GeoNetwork 5 Development & Testing
+This project provides Docker Compose configurations to simplify local development and testing of GeoNetwork 5. It helps set up all necessary services without dealing with infrastructure dependencies manually.
+
+## ⚙️ Development Setup (docker-compose-dev.yml)
+This composition includes:
+
+
+|Service|Functionality|Port|
+|:-----------|:-------------------------|:-----|
+|geonetwork4|GeoNetwork 4 "sidecar" for GN5 (used for harvesting etc.)|8080|
+|database|PostgreSQL — Database backend for GeoNetwork|5432|
+|elasticsearch|Elasticsearch — Search engine backend|9200|
+
+This setup is ideal for developers working on GN5 who need a quick and clean environment.
+
+▶️ Start the stack
+From the /docker directory:
+
+```bash
+docker compose -f docker-compose-dev.yml up -d
```

Comment:
should the "full" command line be shown here:

`docker compose -f docker-compose-dev.yml -f docker-compose-full.yml up -d`